dependencies/pom.xml

```diff
@@ -881,6 +883,12 @@
                 <artifactId>org.eclipse.persistence.jpa</artifactId>
                 <version>${version.lib.eclipselink}</version>
             </dependency>
+            <!-- Needed only to force upgrade to ASM 9.4.0. Once eclipselink is upgraded this can be removed -->
```

Comment:
Maybe indicate _why_ we would want to upgrade ASM to 9.4.0 and why an upgrade of Eclipselink to some other version would let us remove this. I know it, you know it, but maybe a casual reader doesn't know it.